docs/sources/panels-visualizations/query-transform-data/calculation-types/index.md

```diff
@@ -3,6 +3,7 @@ aliases:
   - ../panels/calculation-types/
   - ../panels/calculations-list/
   - ../panels/reference-calculation-types/
```

Comment:
These all need updating because the relative path points to a different page now that this one has moved.

Let me know if you would like me to update them.

Reply:
I would be very grateful!

Reply:
Done in ef127f1148c0a351299ae4637b0c8fa173dcb531

Reply:
thank you!